modules/gdnative/nativescript/SCsub

```diff
@@ -4,7 +4,6 @@ Import('env')
 Import('env_gdnative')
 
 env_gdnative.add_source_files(env.modules_sources, '*.cpp')
-env_gdnative.Append(CPPFLAGS=['-DGDAPI_BUILT_IN'])
```

Comment:
@karroffel This seemed unused according to ripgrep.

Reply:
Yes, I just checked myself, should be okay to remove.